spec/requests/api/v1/invoices_spec.rb

```diff
@@ -134,5 +134,24 @@
         expect(json[:invoices].first[:lago_id]).to eq(invoice.id)
       end
     end
+
+    context 'with payment status param' do
+      let(:invoice) { create(:invoice, customer: customer, payment_status: :succeeded) }
+      let(:invoice2) { create(:invoice, customer: customer, payment_status: :failed) }
+      let(:invoice3) { create(:invoice, customer: customer, payment_status: :pending) }
+
+      before do
+        invoice2
+        invoice3
+      end
+
+      it 'returns invoices with correct payment status' do
+        get_with_token(organization, "/api/v1/invoices?payment_status=pending")
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)